### ISSUE TITLE
[BUG] Alby balance should refresh when receiving funds with in the same wallet but using an external linked wallet

### ISSUE THREAD

**Issue by bitcoinuser** [getAlby/lightning-browser-extension#1674]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I use the same wallet on Alby and Zeus. So I clicked on Alby fav icon on Browser and let it opened. At same time I opened my Zeus wallet and clicked to receive funds. I paid the invoice with Breez.
So I looked to see if Alby balance changed, but no. Only change if I click on fav icon to hide Alby and open again.
But if I switch between outgoing and incoming tabs, I see the new transaction. But Alby balance doesn't change.

### Steps To Reproduce

1. Click on Alby fav icon on Browser and let it opened.
2. Using the same wallet on Alby and Zeus, click on Zeus to receive funds, and pay the invoice
3. See the Alby to see if the balance changes.
4. Click between outgoing and incoming tabs to see the new transaction

### Expected behavior

Alby balance should automatically refresh when new funds arrive. Shouldn't be necessary to click on fav icon again to refresh the balance.
If the transaction is recognized on incoming tab, the Alby balance should change too.

### Alby information

Alby version: 1.18

### Are you working on this?

No

**Comment by escapedcat** [comment]
Context:

Currently we only "pull" the incoming txs "on demand". We do not listen or pull those via i.e. an interval.

**Comment by bitcoinuser** [comment]
> Context:
> 
> Currently we only "pull" the incoming txs "on demand". We do not listen or pull those via i.e. an interval.

Yes, but if the transaction appear on incoming tab, the balance should reflect, right?

**Comment by escapedcat** [comment]
> Yes, but if the transaction appear on incoming tab, the balance should reflect, right?

Ah, uhm, yeah I guess we could trigger an `AccountMenu`-update if the incoming txs are being clicked? Or maybe pull the AccountMenu info via interval?

Smells like `state` @lisabaut :P

**Comment by bitcoinuser** [comment]
In my opinion any action on Alby UI should update the balance.

**Comment by qqqzhch** [comment]
I know little about various connectors. Does the connector support watch？i will collect this information to learn
